### DISCUSSION TITLE
"Failed to setup Minecraft, java.lang.RuntimeException: Failed to remap minecraft" error when trying to build a mod using gradle

### DISCUSSION THREAD

**Discussion by CaptainRexPL** [FabricMC/fabric-api#2510] [Mod Dev Support] [answered]
full log:
"> Configure project :fabric
Fabric Loom: 1.0.4
Found existing cache lock file, rebuilding loom cache. This may have been caused by a failed or canceled build.

Exception in thread "Daemon client event forwarder" java.lang.OutOfMemoryError: Java heap space

> Configure project :fabric
Loom is applying dependency data manually to publications instead of using a software component (from(components["java"])). This is deprecated.
Deprecated Loom APIs were used in this build, making it incompatible with future versions of Loom. Use Gradle warning modes to control the verbosity of the warnings.

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':fabric'.
> Failed to setup Minecraft, java.lang.RuntimeException: Failed to remap minecraft

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 52s"
loom version: 1.0.4
I tried changing org.gradle.jvmargs to 6GB instead of 1GB set by default and the error is still the same.

**Comment by 12122234** [comment, marked as answer]
Try ./gradlew --debug

**Reply by CaptainRexPL** [reply]
I checked with debug and I just forgot that I'm changing this only in gradle.properties that was in "fabric" folder not in global gradle.properties. I changed it and now it works fine.

**Reply by 12122234** [reply]
Good!
